src/components/05_pages/Content/Content.js

```diff
@@ -0,0 +1,218 @@
+    contentTypes: PropTypes.objectOf(
+      PropTypes.shape({
+        name: PropTypes.string.isRequired,
+        description: PropTypes.string.isRequired,
+      }),
+    ).isRequired,
+    requestContent: PropTypes.func.isRequired,
+    requestContentTypes: PropTypes.func.isRequired,
+    nodes: PropTypes.arrayOf(PropTypes.object),
+  };
+  static defaultProps = {
+    nodes: [],
+  };
+  state = {
+    contentTypes: [],
+    status: null,
+  };
+  componentDidMount() {
+    this.props.requestContentTypes();
+    this.props.requestContent();
+  }
+  render = () => (
+    <Fragment>
+      <div className={styles.filters}>
+        <TextField
+          label="Title"
+          placeholder="Title"
+          onChange={e => {
+            this.setState({ title: e.target.value }, () => {
+              this.props.requestContent(this.state);
+            });
+          }}
+          margin="normal"
+        />
+
+        <FormControl className={styles.formControl}>
+          <InputLabel htmlFor="select-multiple-checkbox">
+            Content Type
+          </InputLabel>
+          <Select
+            multiple
+            value={this.state.contentTypes}
+            onChange={e => {
+              this.setState({ contentTypes: e.target.value }, () => {
+                this.props.requestContent(this.state);
+              });
+            }}
+            input={<Input id="select-multiple-checkbox" />}
+            renderValue={selected => (
+              <div className={styles.chips}>
+                {selected.map(value => (
+                  <Chip
+                    key={value}
+                    label={this.props.contentTypes[value].name}
+                    className={styles.chip}
+                  />
+                ))}
+              </div>
+            )}
+          >
+            {Object.keys(this.props.contentTypes).map(type => (
+              <MenuItem key={type} value={type}>
+                <Checkbox
+                  checked={this.state.contentTypes.indexOf(type) > -1}
+                />
+                <ListItemText primary={this.props.contentTypes[type].name} />
+              </MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+
+        <FormControl className={styles.formControl}>
+          <InputLabel htmlFor="status">Status</InputLabel>
+          <Select
+            value={this.state.status || ''}
+            onChange={e => {
+              this.setState({ status: e.target.value }, () => {
+                this.props.requestContent(this.state);
+              });
+            }}
+            input={<Input name="status" id="status" />}
+            autoWidth
+          >
+            <MenuItem value="">
+              <em>Any</em>
+            </MenuItem>
+            <MenuItem value="published">Published</MenuItem>
+            <MenuItem value="unpublished">Unpublished</MenuItem>
+          </Select>
+        </FormControl>
+      </div>
+      <Paper>
+        <Table>
```

Comment:
Should we show a loading animation at least on the initial page load?

Reply:
@lauriii I removed the LoadingBar from this page as I think we should move it to the Default template, but didn't want to start exploring that here

Reply:
@justafish makes sense! Let's not block this PR with that, but we should open an issue for that anyway! 🙂